Source/WebCore/inspector/agents/InspectorDOMAgent.cpp

```diff
@@ -3095,4 +3106,84 @@ Protocol::ErrorStringOr<void> InspectorDOMAgent::setAllowEditingUserAgentShadowT
     return { };
 }
 
+Protocol::ErrorStringOr<Ref<Protocol::DOM::MediaStats>> InspectorDOMAgent::getMediaStats(Protocol::DOM::NodeId nodeId)
+{
+    Protocol::ErrorString errorString;
+
+    Element* element = assertElement(errorString, nodeId);
+    if (!element)
+        return makeUnexpected(errorString);
+
+    if (!element->isMediaElement())
```

Comment:
Could use `dynamicDowncast<HTMLMediaElement>` on `element` and have an early return if the value is `nullptr` here. That way you can combine the `isMediaElement()` test here and the `downcast<>` below.